frontend/src/i18n/locales/de.yaml

```diff
@@ -261,6 +261,12 @@ manage:
           dummy: Serie auswählen
           invalid: Bitte eine Serie auswählen
 
+      event:
+        event:
```

Comment:
Why two nested `event`? Goes for the series above too. Just pull the doubly nested things one level higher?

Reply:
Because there might be other fields, like for series. The first event/series says were are talking about an event/series block. The second is for things relating to the series/event selection box.

Now that I think about it, the first level should probably be called `video` in this case ... 🤔

Reply:
I would still not nest those things, but put them directly under the top level `event` or `video`, but whatever.